pages/search.js

```diff
@@ -61,7 +76,7 @@ const formatError = (error) => {
     errorString += ` (${error.errno})`
   }
   if (errorString === '') {
-    errorString = JSON.stringify(error)
+    errorString = JSON.stringify(error, getCircularReplacer())
```

Comment:
Why are we getting circular references here? Maybe we should open an issue on zeit/next about it?